tools/run_cargo_fmt.sh

```diff
@@ -64,7 +64,7 @@ done
 # Find folders with Cargo.toml files in them and check them (avoids matching this script!)
 for f in $(find . | grep Cargo.toml); do
 	pushd $(dirname $f) > /dev/null
-	if $(git grep -q 'use .*\*;'); then
+	if $(git grep -q 'use .*\*;' -- ':!src/macros.rs'); then
```

Comment:
Why does macros.rs need to be excluded?

I prefer to trust in rustfmt, but there are ways to be very specific about what fustfmt should ignore.

Reply:
The confusion part here is that this `run_rust_fmt.sh` script also handles looking for wildcard import patterns (the part that I changed here), which has nothing to do with formatting. I'd suggest to split the script in 2 files to make this clearer.

Reply:
Moved that part to a standalone script.